jobs/build/advisories/Jenkinsfile

```diff
@@ -202,6 +214,9 @@ node {
                     if (params.ENABLE_AUTOMATION) {
                         cmd += "sed -e 's/freeze_automation:.*/freeze_automation: no/' -i group.yml ; "
                     }
+                    if (params.ENABLE_ARCH_OVERRIDE) {
+                        cmd += "gawk --include inplace '/^#? *arches_override:$/{flag=1;print \$NF;next}; NF<=1{flag=0}; flag {print $(NF-1), \$NF;next}; 1'"
```

Comment:
Make sure to test this :)
Embedding code with `$` in a GString has always seemed problematic. Since nothing here is using interpolation, why not use a `'''` string?
```suggestion
                        cmd += '''gawk --include inplace '/^#? *arches_override:$/{flag=1;print $NF;next}; NF<=1{flag=0}; flag {print $(NF-1), $NF;next}; 1' '''
```